meta-multimedia: add tinycompress recipe

qti-sbojja opened the pull request:
Add BitBake recipe for tinycompress version 1.2.13 to meta-multimedia. Tinycompress is a minimal library for interacting with ALSA compressed audio offload APIs.

kraj commented:
> Thanks @kraj for the review and suggestions. Addressed the comments.

Commit message is not as per [recipe guidelines](https://docs.yoctoproject.org/dev/contributor-guide/recipe-style-guide.html#recipe-style-guide) - Please follow then in future since I have fixed it this time in what I cherry picked

qti-sbojja commented:
> > Thanks @kraj for the review and suggestions. Addressed the comments.
> 
> Commit message is not as per [recipe guidelines](https://docs.yoctoproject.org/dev/contributor-guide/recipe-style-guide.html#recipe-style-guide) - Please follow then in future since I have fixed it this time in what I cherry picked

Sure, thankyou.

kraj commented:
merged to master with c0226393ba
